torch/_cuda_sanitizer/_sanitizer.py

```diff
@@ -0,0 +1,490 @@
+import enum
```

Comment:
nit: if you under-scored the parent module, you don't have to underscore this module. But I'd probably suggest putting this in `torch.cuda._sanitizer`, which makes use of the existing namespace torch.cuda

Reply:
Done.